src/services/penaltyGroups.js

```diff
@@ -97,8 +115,7 @@ export default class PenaltyGroup {
 			if (!anyOutstanding) {
 				// Update penalty group payment status if all penalties have been paid
 				penaltyGroup.PaymentStatus = paymentStatus;
-				// TODO: Implement 'Enabled' flag (indicates whether or not a penalty group was deleted)
-				penaltyGroup.Hash = hashToken(id, penaltyGroup, true);
+				penaltyGroup.Hash = hashToken(id, penaltyGroup, paymentStatus);
```

Comment:
how comes we're using paymentStatus here as the 3rd arg and penGrp.Enabled below?

Reply:
Good spot, fixing!